source/docs/articles/sites/migrate/manual-site-import.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Migrate to Pantheon: Manual Site Import
+description: Learn how to import a Drupal or WordPress site into Pantheon outside of the Importer Tool.
+keywords: import, importing site, pantheon, new site, large site, distro, upstream, git history
+---
+Manually import your site to Pantheon outside of the provided [Importer Tool](/docs/articles/sites/migrate/#plan-the-import) when any of the following apply:
+
+* **Large Site Archives**: Site archive is greater than the automated import limits (100MB for direct file upload or 500MB for URL upload).
+* **Custom Upstream**: Site should receive updates based on an upstream other than vanilla Drupal or WordPress (e.g Panopoly, or your agency's customized WordPress).
+* **Preserve Git History**: Site's existing Git commit history should be retained
+
+## Requirements
+
+* [Git](/docs/articles/local/starting-with-git/)
+* [Rsync or SFTP Client](https://pantheon.io/docs/articles/local/rsync-and-sftp/)
+* [MySQL Client](https://pantheon.io/docs/articles/local/accessing-mysql-databases/)
+
+## Create a New Pantheon Site and Start from Scratch
+
+From your Pantheon Dashboard:
+
+* Choose **Create a new site**.
+* Name your site.
+* Select **Start from scratch**, and choose your starting codebase.
+
+Starting from scratch allows your site to connect to that upstream so you can later [apply upstream updates](/docs/articles/sites/code/applying-upstream-updates/) from your Dashboard with one click.
+
+## Import the Codebase
+
+**Codebase** - all executable code, including core, custom and contrib modules or plugins, themes, and libraries.
+
+As long as you've chosen the same codebase (Drupal 7, Commerce Kickstart, etc.) as the starting point of your Pantheon site, you can use Git to import your existing code and commit history. If you don’t have a Git version controlled codebase, the following will still work.
+
+1. Go to your code directory within your terminal.
+2. If your existing site code is not version controlled with Git, run: ```git init```
+3. From the Site Dashboard, go to the **Dev** environment.
+4. Switch the site's connection mode from SFTP to Git.
+5. Get the upstream's Git connection string:

```

Comment:
If they have already created the site, would it not be simpler to use the site repo in steps 5-6? Something along the lines of:
1. git clone ssh:codeserver-uuid.git site
2. merge code locally (add custom and contrib to repo) and commit.
3. Push

Thoughts?